pgcli/packages/parseutils.py

```diff
@@ -149,12 +152,22 @@ def extract_tables(sql):
     return list(extract_table_identifiers(stream))
 
 def find_prev_keyword(sql):
+    """ Find the last sql keyword in an SQL statement
+
+    Returns the value of the last keyword, and the text of the query with
+    everything after the last keyword stripped
+    """
     if not sql.strip():
-        return None
+        return None, ''
 
-    for t in reversed(list(sqlparse.parse(sql)[0].flatten())):
+    parsed = sqlparse.parse(sql)[0]
+    for t in reversed(list(parsed.flatten())):
         if t.is_keyword or t.value == '(':
-            return t.value
+            idx = parsed.token_index(t)
+            text = ''.join(tok.value for tok in parsed.tokens[:idx+1])
+            return t.value, text
```

Comment:
I don't understand why this change was made. I see that it's used to update the `text_before_cursor` in line 230 in sqlcompletion.py. But I'm lacking the context as to why.